auto-complete.js

```diff
@@ -5,40 +5,41 @@
     License: http://www.opensource.org/licenses/mit-license.php
 */
 
-
+const { remmoveSuggestionFromCache } = require('./utils/cache');
+const { removeQueryFromLocalStorage, addQueryToLocalStorage, getQueriesFromLocalStorage, removeDuplicatedQueries } = require('./utils/localStorage');
```

Comment:
It seems that this project does not transpile ES2015+ code to ES5.
You have to setup babel transpilation or drop all usages of ES2015+ syntax in your code, like `const`, `let`, arrow function and object destructuring.

Reply:
I dropped usages of ES2015+ :+1: .
This project is small babel it seems unnecessary for now